test/deploy/rackhd_ha_install.py

```diff
@@ -0,0 +1,217 @@
+            dhcp_conf.close()
+            # copy file to ORA
+            fit_common.scp_file_to_ora('dhcpd.conf', vmnum)
+            os.remove('dhcpd.conf')
+            self.assertEqual(fit_common.remote_shell('cp dhcpd.conf /etc/dhcp/', vmnum=vmnum)['exitcode'],
+                             0, "DHCP Config failure.")
+
+    def create_hosts(self):
+        hosts_conf = open('hosts-conf', 'w')
+        for vmnum in range(1, numvms + 1):
+            line = '172.31.128.{}\tnode{}\n'.format(vmnum, vmnum)
+            hosts_conf.write(line)
+        hosts_conf.close()
+
+    def create_node_list(self):
+        result = ""
+        sb_net = self.get_southbound_network()
+        if sb_net:
+            for vmnum in range(1, numvms + 1):
+                result += "   node {{\n      ring0_addr: {1}.{0}\n      nodeid: {0}\n   }}\n".format(vmnum, sb_net)
+        return result
+
+    def find_southbound(self, httpEndpoints):
+        for i in httpEndpoints:
+            if i["routers"] == "southbound-api-router":
+                return i
+        return None
+
+    def get_template_file(self, file_name):
+        template_folder = './config_templates'
+        env = Environment(loader=FileSystemLoader(template_folder))
+        return env.get_template(file_name)
+
+    def get_southbound_network(self):
+        endpoints = fit_common.fitrackhd()['httpEndpoints']
+        southbound = self.find_southbound(endpoints)
+        if southbound and 'address' in southbound:
+            address = southbound["address"]
+            addrsplit = address.split('.')
+            return("{}.{}.{}".format(addrsplit[0], addrsplit[1], addrsplit[2]))
+        return None
+
+    def create_corosync_config(self):
+        template = self.get_template_file("corosync_config")
+        rendered = template.render(node_list=self.create_node_list())
+        return rendered
+
+    def test03_install_hosts_config(self):
+        sb_net = self.get_southbound_network()
+        self.assertIsNotNone(sb_net, "Could not find southbound address")
+
+        for vmnum in range(1, numvms + 1):
+            self.create_hosts()
+            # copy file to ORA
+            fit_common.scp_file_to_ora('hosts-conf', vmnum)
+            # Clean out the previous entries to be idempotent
+            command = "grep -v {} /etc/hosts > hosts".format(sb_net)
+            self.assertEqual(fit_common.remote_shell(command, vmnum=vmnum)['exitcode'],
+                             0, "Hosts Config failure.")
+            # Add the new entries
+            self.assertEqual(fit_common.remote_shell('cat hosts-conf >> hosts', vmnum=vmnum)['exitcode'],
+                             0, "Hosts Config failure.")
+            # Move the new file into place
+            self.assertEqual(fit_common.remote_shell('mv hosts /etc/hosts', vmnum=vmnum)['exitcode'],
+                             0, "Hosts Config failure.")
+        os.remove('hosts-conf')
+
+    def test04_install_hostname_config(self):
```

Comment:
same comment - add the @depend(after=xxx) if you want the test to be skipped.  Same with subsequent test05, 06, 07.